plugins.json

```diff
@@ -34641,6 +34641,140 @@
+    "download_url": "https://plugins.releases.mattermost.com/release/mattermost-plugin-mscalendar-v1.5.0.tar.gz",
+    "release_notes_url": "https://mattermost.com/pl/mattermost-plugin-mscalendarreleases/tag/v1.5.0",
```

Comment:
There is this typo again - Is this a bug in the script we are running?

Reply:
@marianunez @nevyangelova  The problem is here:  https://github.com/mattermost/mattermost-plugin-mscalendar/blob/f1da864f0805a9516f7eb6fea76d9db6fcdcc7ee/build/manifest/main.go#L146

Unfortunately, that build manifest code is copied to all repos and not imported from a central spot.